app/src/main/java/com/babylon/wallet/android/presentation/ui/composables/card/DappCard.kt

```diff
@@ -27,7 +27,7 @@ fun DappCard(
     modifier: Modifier = Modifier,
     dApp: DApp,
     showChevron: Boolean = true,
-    elevation: Dp = 8.dp
+    elevation: Dp = 2.dp
```

Comment:
This was done to change appearance of the shadow? I created "default" shadow modifier in one of the PRs, maybe it will makes sense to align all cards shadow to use exact same logic when we start merging the PRs

Reply:
Right, makes sense.